packages/ddl-shim/src/upgrade.ts

```diff
@@ -584,22 +593,34 @@ class DDLUpgrade {
         };
     }
 
-    type2chartType(chartType: DDL1.VisualizationType): { chartType: string, __class: string } {
+    type2chartType(chartType: DDL1.VisualizationType | string): { chartType: string, __class: string } {
+        console.log("chartType", chartType);
         switch (chartType) {
+            case "AM_LINE":
+            case "C3_LINE":
+            case "GOOGLE_LINE":
             case "LINE":
                 return { chartType: "Line", __class: "chart_Line" };
             case "BUBBLE":
                 return { chartType: "Bubble", __class: "chart_Bubble" };
+            case "AM_PIE":
+            case "C3_PIE":
+            case "GOOGLE_PIE":
             case "PIE":
                 return { chartType: "Pie", __class: "chart_Pie" };
+            case "AM_BAR":
+            case "C3_BAR":
+            case "GOOGLE_BAR":
             case "BAR":
```

Comment:
"BAR" is actually a roxie keyword and does translate to COLUMN...

Reply:
Ah okay, so should I change this back to COLUMN+chart_Column?

Reply:
Yes - actually I suspect this is not the correct place for any of the "chartTypes" as these are the Roxie DDL keywords.